bech32/cov_report.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Please remove this file.  We really should remove it from the other packages as well.  It is no longer supported since `go test` provides coverage nowadays.